runtime/interpreter/errors.go

```diff
@@ -80,16 +85,7 @@ func (e PositionedError) Error() string {
 	return e.Err.Error()
 }
 
-// ExternalError is an error that occurred externally.
-// It contains the recovered value.
-//
-type ExternalError struct {
-	Recovered any
-}
-
-func (e ExternalError) Error() string {
-	return fmt.Sprint(e.Recovered)
-}
+func (e PositionedError) IsUserError() {}
```

Comment:
Here and for the other errors below:

```suggestion
func (PositionedError) IsUserError() {}
```